pkg/service/kubeconfig_settings_test.go

```diff
@@ -0,0 +1,164 @@
+	db, mock := getDB(t)
+	defer db.Close()
+
+	ps := NewKubeconfigSettingService(db)
+
+	uuuid := uuid.New().String()
+	ouuid := uuid.New().String()
+	acuuid := uuid.UUID.String(uuid.New())
+	validity_seconds := 300
+	disable_web_kubectl := true
+	disable_cli_kubectl := true
+
+	mock.ExpectQuery(`SELECT "ks"."id", "ks"."organization_id", "ks"."partner_id", "ks"."account_id", "ks"."scope", "ks"."validity_seconds", "ks"."created_at", "ks"."modified_at", "ks"."deleted_at", "ks"."enforce_rsid", "ks"."disable_all_audit", "ks"."disable_cmd_audit", "ks"."is_sso_user", "ks"."disable_web_kubectl", "ks"."disable_cli_kubectl", "ks"."enable_privaterelay", "ks"."enforce_orgadmin_secret_access" FROM "sentry_kubeconfig_setting" AS "ks" WHERE \(organization_id = '` + ouuid + `'\) AND \(account_id = '` + acuuid + `'\)`).
+		WithArgs().WillReturnRows(sqlmock.NewRows([]string{"id", "organization_id", "account_id", "validity_seconds", "disable_web_kubectl", "disable_cli_kubectl"}))
+
+	kss := &sentry.KubeconfigSetting{Id: uuuid, OrganizationID: ouuid, AccountID: acuuid, ValiditySeconds: int64(validity_seconds), DisableWebKubectl: disable_web_kubectl, DisableCLIKubectl: disable_cli_kubectl}
+
+	kss, err := ps.Get(context.Background(), ouuid, acuuid, false)
+	if err == nil {
+		t.Fatal("kubeconfig setting found for invalid user")
+		t.Log(kss.GetId())
+	}
+}
+
+func TestUpdateKubeconfigSetting(t *testing.T) {
+	db, mock := getDB(t)
+	defer db.Close()
+
+	ps := NewKubeconfigSettingService(db)
+
+	uuuid := uuid.New().String()
+	ouuid := uuid.New().String()
+	acuuid := uuid.UUID.String(uuid.New())
+	validity_seconds := 300
+	disable_web_kubectl := true
+	disable_cli_kubectl := true
+
+	kss := &sentry.KubeconfigSetting{Id: uuuid, OrganizationID: ouuid, AccountID: acuuid, ValiditySeconds: int64(validity_seconds), DisableWebKubectl: disable_web_kubectl, DisableCLIKubectl: disable_cli_kubectl}
+
+	mock.ExpectBegin()
+
+	mock.ExpectQuery(`SELECT "ks"."id", "ks"."organization_id", "ks"."partner_id", "ks"."account_id", "ks"."scope", "ks"."validity_seconds", "ks"."created_at", "ks"."modified_at", "ks"."deleted_at", "ks"."enforce_rsid", "ks"."disable_all_audit", "ks"."disable_cmd_audit", "ks"."is_sso_user", "ks"."disable_web_kubectl", "ks"."disable_cli_kubectl", "ks"."enable_privaterelay", "ks"."enforce_orgadmin_secret_access" FROM "sentry_kubeconfig_setting" AS "ks" WHERE \(organization_id = '` + ouuid + `'\) AND \(account_id = '` + acuuid + `'\)`).
+		WithArgs().WillReturnRows(sqlmock.NewRows([]string{"id", "organization_id", "account_id"}).AddRow(uuuid, ouuid, acuuid))
+
+	mock.ExpectExec(`UPDATE "sentry_kubeconfig_setting" AS "ks" SET .*, validity_seconds = 300, enforce_rsid = FALSE, is_sso_user = FALSE, disable_web_kubectl = TRUE, disable_cli_kubectl = TRUE, enable_privaterelay = FALSE, enforce_orgadmin_secret_access = FALSE WHERE \(organization_id = '` + ouuid + `'\) AND \(account_id = '` + acuuid + `'\) AND \(is_sso_user= FALSE\)`).
```

Comment:
We should probably use the variables you have define up for things like `validity_seconds` and other here instead of hardcoding the values.